src/lib/y2storage/planned/can_be_mounted.rb

```diff
@@ -37,6 +37,12 @@ module CanBeMounted
       # Initializations of the mixin, to be called from the class constructor.
       def initialize_can_be_mounted
       end
+
+      # Whether the device will be mounted as root
+      # @return [Boolean]
+      def root?
+        mount_point == "/"
```

Comment:
We already have `Y2Storage::Mountable::ROOT_PATH`. Should we start to use that constant instead of literal "/"?